kitsune/sumo/jinja2/includes/common_macros.html

```diff
@@ -51,6 +51,97 @@
 </div>
 {% endmacro %}
 
+{% macro browse_by_topic(product=True) %}
+  {% set row_one_left_slug="settings" %}
+  {% set row_one_right_slug="installation-and-updates" %}
+  {% set row_two_left_slug="email-and-messaging" %}
+  {% set row_two_right_slug="performance-and-connectivity" %}
+  {% set row_three_left_slug="privacy-and-security" %}
+  {% set row_three_right_slug="browse" %}
+
+  {% if product %}
+    {% set link_name ="main-menu.topic-home" %}
+    {% set row_one_left_url = url('products.topic_documents', topic_slug=row_one_left_slug ) %}
+    {% set row_one_right_url = url('products.topic_documents', topic_slug=row_one_right_slug ) %}
+    {% set row_two_left_url = url('products.topic_documents', topic_slug=row_two_left_slug ) %}
+    {% set row_two_right_url = url('products.topic_documents', topic_slug=row_two_right_slug ) %}
+    {% set row_three_left_url = url('products.topic_documents', topic_slug=row_three_left_slug ) %}
+    {% set row_three_right_url = url('products.topic_documents', topic_slug=row_three_right_slug ) %}
+
+  {% else %}
+    {% set link_name ="main-menu.support-forum-home" %}
+    {% set row_one_left_url = url('questions.list_by_topic', topic_slug=row_one_left_slug ) %}
+    {% set row_one_right_url = url('questions.list_by_topic', topic_slug=row_one_right_slug ) %}
+    {% set row_two_left_url = url('questions.list_by_topic', topic_slug=row_two_left_slug ) %}
+    {% set row_two_right_url = url('questions.list_by_topic', topic_slug=row_two_right_slug ) %}
+    {% set row_three_left_url = url('questions.list_by_topic', topic_slug=row_three_left_slug ) %}
+    {% set row_three_right_url = url('questions.list_by_topic', topic_slug=row_three_right_slug ) %}
+  {% endif %}
+
+  <ul class="mzp-c-menu-item-list sumo-nav--sublist">
+    <li>
+      <a href="{{ row_one_left_url }}"
+          data-event-name="link_click"
+          data-event-parameters='{
+          "link_name": {{ link_name }},
```

Comment:
The `{{ link_name }}` needs to be wrapped in quotes on this one and all that follow.
```suggestion
          "link_name": "{{ link_name }}",
```